src/jdk.jpackage/share/classes/jdk/jpackage/internal/DeployParams.java

```diff
@@ -111,7 +111,7 @@ static void validateName(String s, boolean forApp)
             if (forApp) {
                 return;
             } else {
-                throw new PackagerException(exceptionKey, s);
+                throw new PackagerException(exceptionKey);
```

Comment:
Wouldn't it be better to do throw new PackagerException(exceptionKey, "null"); so the message looks more understandable later?

Reply:
isn't that exactly the same ?